modules/rosa-hcp-upgrading-cli-machinepool.adoc

```diff
@@ -0,0 +1,109 @@
++
+[source,terminal]
+----
+$ rosa list upgrade --cluster <cluster-name> --machinepool <machinepool_name>
+----
++
+The command returns a list of available updates, including the recommended version.
++
+.Example output
++
+[source,terminal]
+----
+VERSION  NOTES
+4.14.5   recommended
+4.14.4
+4.14.3
+----
++
+[IMPORTANT]
+====
+Do not upgrade your machine pool to a version higher than your control plane. If you want to move to a higher version, upgrade the control plane to that version first.
+====
+//Is it even possible to do this? Will a higher version display? Can you specify a higher version even if it doesn't display?
+
+. Verify the upgrade behavior of the machine pools you intend to upgrade by running the following command:
++
+[source,terminal]
+----
+$ rosa describe machinepool --cluster=<cluster_name_or_id> <machine_pool_name> 
+----
++
+.Example output
+[source,terminal]
+----
+Replicas: 5
+Node drain grace period:   30 minutes
+
+Management upgrade:
+- Type: Replace
+- Max surge: 20%
+- Max unavailable: 5%
+----
++
+In the example, these settings allow the machine pool to provision one excess node (`max-surge` of 20% of `replicas`) and to have up to one node unavailable (`max-unavailable` of 20% of `replicas`) during an upgrade. This machine pool can therefore upgrade two nodes at a time, by provisioning one new node in excess of the replica count, and by making one node unavailable and replacing it. Node upgrades may be delayed by up to 30 minutes (`node-drain-grace-period` of 30 minutes) if necessary to protect workloads that have a pod disruption budget.
+
+. Upgrade one or more of your machine pools by running the following command:
++
+[source,terminal]
+----
+$ rosa upgrade machinepool -c <cluster_name> <first_machine_pool_id> <second_machine_pool_id> [--schedule-date=<yyyy-mm-dd> --schedule-time=<HH:mm>] --version <version_number>
+----
++
+Multiple machine pools can be upgraded simultaneously. You can schedule machine pool upgrades individually or schedule multiple upgrades in a single command.
+
+** To schedule the immediate upgrade of a specific machine pool on your cluster, run the following command:
++
+[source,terminal]
+----
+$ rosa upgrade machinepool -c <cluster_name> <your_machine_pool_id> --version <version_number>
+----
++
+Your machine pool is scheduled for immediate upgrade, which initiates a rolling replacement of all nodes in the specified machine pool.
+
+** To schedule an upgrade of multiple machine pools to start at a future date, run the following command:
++
+[source,terminal]
+----
+$ rosa upgrade machinepool -c <cluster_name> <first_machine_pool_id> <second_machine_pool_id> --schedule-date=<yyyy-mm-dd> --schedule-time=<HH:mm> --version <version_number>
+----
++
+Your machine pools are scheduled to begin an upgrade at the specified time and date in Coordinated Universal Time (UTC). This will initiate a rolling replacement of all nodes in the specified machine pools, beginning at the specified time.
```

Comment:
Suggestion: 
s/This will initiate a rolling replacement.../This initiates a rolling replacement...